base/sysinfo.jl

```diff
@@ -532,7 +533,26 @@ including e.g. a WebAssembly JavaScript embedding in a web browser.
 """
 isjsvm(os::Symbol) = (os === :Emscripten)
 
-for f in (:isunix, :islinux, :isbsd, :isapple, :iswindows, :isfreebsd, :isopenbsd, :isnetbsd, :isdragonfly, :isjsvm)
+"""
+    Sys.detectwsl([os])
```

Comment:
Perhaps `Sys.iswsl()` for consistency with the other platform detection functions?

Reply:
This is actually on purpose, as `detectwsl` detects at runtime, so names of the form `is_platform` are not used.
See discuss in #36354